app/code/Magento/InventorySales/etc/events.xml

```diff
@@ -6,9 +6,6 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Event/etc/events.xsd">
-    <event name="website_save_after">
-        <observer name="assign_website_to_default_stock" instance="Magento\InventorySales\Observer\Website\AssignWebsiteToDefaultStock"/>
-    </event>
     <event name="website_delete_after">
```

Comment:
Looks like we also need to update delete logic `website_delete_after`

Reply:
An observer after delete seems to be an efficient way and seems to be working, do you prefer to pluginize it?

Reply:
ref: https://github.com/magento-engcom/msi/pull/1356